README.md

```diff
@@ -11,7 +11,7 @@
 * [Help Improving the Documents](#help-improving-the-documents)
 * [Website Changes](#website-changes)
 
-The [Kata Containers](https://github.com/kata-containers)
+The [Kata Containers\*](https://github.com/kata-containers)
```

Comment:
Generally the Kata project has not used the `*` notation for 'copyright of others' in its docs. Probably best not to start adding them, unless you intend to add them to *all* the docs :-)

Reply:
Agree.